examples/nxp/rt1060-evk-make-baremetal-builtin/main.c

```diff
@@ -31,16 +32,57 @@ static void timer_fn(void *arg) {
            ifp->ndrop, ifp->nerr));
 }
 
+#ifdef MQTT_DASHBOARD
+bool hal_gpio_write(int pin, bool status) {  // For MQTT dashboard HAL
+  return (pin != status);
+}
+bool hal_gpio_read(int pin) {  // For MQTT dashboard HAL
+  return (pin != 0);
+}
+int hal_led_pin(void) {
+  return (int) 0;
+}
+uint64_t mg_now(void) {
+  return mg_millis();
+}
+#endif
+
 int main(void) {
-  gpio_output(LED);               // Setup blue LED
+  gpio_output(LED);               // Setup green LED
   uart_init(UART_DEBUG, 115200);  // Initialise debug printf
   ethernet_init();                // Initialise ethernet pins
-  MG_INFO(("Starting, CPU freq %g MHz", (double) SystemCoreClock / 1000000));
```

Comment:
Why move it ? All examples have it there

Reply:
My bad, I didn't notice... I'll put it back